apps/api/src/app/environments/environment.prod.ts

```diff
@@ -31,7 +31,7 @@ export default {
   },
   healthInsurance: {
     wsdlUrl: process.env.HEALTH_INSURANCE_XROAD_WSDLURL,
-    baseUrl: process.env.XROAD_BASE_PATH_WITH_ENV,
+    baseUrl: process.env.XROAD_BASE_PATH,
```

Comment:
For anyone else wondering why this is being changed, [here's a relevant discussion](https://github.com/island-is/island.is/pull/3095#discussion_r586327870).